Stripe/STPPaymentMethodTableViewCell.h

```diff
@@ -7,12 +7,20 @@
 //
 
 #import <UIKit/UIKit.h>
-#import "STPPaymentMethod.h"
-#import "STPTheme.h"
+
+@class STPTheme;
+
+@protocol STPPaymentMethod;
+
+NS_ASSUME_NONNULL_BEGIN
 
 @interface STPPaymentMethodTableViewCell : UITableViewCell
 
-- (void)configureWithPaymentMethod:(id<STPPaymentMethod>)paymentMethod selected:(BOOL)selected theme:(STPTheme *)theme;
+- (instancetype)initWithStyle:(UITableViewCellStyle)style reuseIdentifier:(nullable NSString *)reuseIdentifier;
```

Comment:
hmm yeah we don't need this declaration. i was playing around with the `nullability` annotations but didn't get anywhere with it.